fontir/src/ir.rs

```diff
@@ -362,38 +362,117 @@ pub enum GlobalMetric {
     SuperscriptYSize,
 }
 
+impl GlobalMetric {
+    /// Return the 4-byte tag used to represent a global metric in the `MVAR` table.
+    ///
+    /// `None` if this metric is not associated with an `MVAR` value tag, or if
+    /// we don't support it yet (e.g. vertical typesetting metrics).
+    ///
+    /// <https://learn.microsoft.com/en-us/typography/opentype/spec/mvar#value-tags>
+    pub fn mvar_tag(&self) -> Option<Tag> {
+        // We support the same subset of the metrics defined in the spec
+        // as fonttools does (except vertical ones which we don't support yet):
+        // https://github.com/fonttools/fonttools/blob/0c5cb3b/Lib/fontTools/varLib/mvar.py
+        match self {
+            GlobalMetric::Os2TypoAscender => Some(Tag::new(b"hasc")),
+            GlobalMetric::Os2TypoDescender => Some(Tag::new(b"hdsc")),
+            GlobalMetric::Os2TypoLineGap => Some(Tag::new(b"hlgp")),
+            GlobalMetric::Os2WinAscent => Some(Tag::new(b"hcla")),
+            GlobalMetric::Os2WinDescent => Some(Tag::new(b"hcld")),
+            // TODO: support vertical global metrics
```

Comment:
Suggest filing an issue and citing it, e.g. // TODO(#xyz): support vertical global metrics

Reply:
#668